setup.py

```diff
@@ -9,6 +11,15 @@
 with open('README.rst', 'r') as f:
     readme = f.read()
 
+if os.system('curl --version | grep -i nss') == 0:
```

Comment:
Other solution would be replace the shell pipeline using the approach mentioned at [1], but the solution mentioned above is simpler.

[1] https://docs.python.org/2/library/subprocess.html#replacing-shell-pipeline